letsencrypt/README.rst

```diff
@@ -97,6 +114,13 @@ an upstream for your odoo instance and do something like::
         proxy_pass    http://yourodooupstream;
     }
 
+If you're using a multi-database installation (with or without dbfilter option)
```

Comment:
I don't think this is true. We run quite some instances with multiple databases, and there the dbfilter (which comes from a header in our case, but that should be the same mechanism) works fine.